debian/xenial/kubeadm/debian/postinst

```diff
@@ -22,8 +22,8 @@ case "$1" in
         # because kubeadm package adds kubelet drop-ins, we must daemon-reload
         # and restart kubelet now. restarting kubelet is ok because kubelet
         # postinst configure step auto-starts it.
-        systemctl daemon-reload
-        systemctl restart kubelet
+        systemctl daemon-reload >/dev/null || true
```

Comment:
should these be `2>/dev/null`?

Reply:
`>/dev/null` or `2>/dev/null` - it is doesn't matter, because systemctl will return nothing in good case, in bad case systemctl will return dbus error (which should be ignored for both commands)